roles/mysql/templates/mysql56.repo.j2

```diff
@@ -0,0 +1,8 @@
+[mysql56-community]
+name=MySQL 5.6 Community Server
+baseurl=http://repo.mysql.com/yum/mysql-5.6-community/el/7/x86_64/
+enabled=1
+gpgcheck=0
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-mysql
```

Comment:
same question as above. Example or actual file?

Reply:
reply is same as above